src/codegen/sys/lib_.rs

```diff
@@ -97,7 +97,7 @@ fn generate_extern_crates(w: &mut dyn Write, env: &Env) -> Result<()> {
 
 fn get_extern_crate_string(library: &ExternalLibrary) -> String {
     format!(
-        "extern crate {}_sys as {};\n",
+        "use {}_sys as {};\n",
```

Comment:
You could directly change that name inside the `Cargo.toml` with 2018 edition:

```toml
ffi = { package = "glib-sys", version = "..." }
```

for example. That seems like a nicer solution. Same for these imports in the non-sys crates.

Reply:
I approve, however I won't do it here. ;)

Reply:
Create an issue then :)

Reply:
https://github.com/gtk-rs/gir/issues/998